samples/nrf5340/empty_app_core/prj.conf

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+CONFIG_MULTITHREADING=n
+CONFIG_NUM_PREEMPT_PRIORITIES=0
+CONFIG_SYS_CLOCK_EXISTS=n
+CONFIG_ARM_MPU=n
+CONFIG_SIZE_OPTIMIZATIONS=y
+CONFIG_I2C=n
+CONFIG_WATCHDOG=n
+CONFIG_GPIO=n
+CONFIG_PINMUX=n
+CONFIG_SPI=n
+CONFIG_SERIAL=n
+CONFIG_FLASH=n
+CONFIG_SYS_POWER_MANAGEMENT=n
+CONFIG_DYNAMIC_INTERRUPTS=n
+CONFIG_IRQ_OFFLOAD=n
+CONFIG_THREAD_STACK_INFO=n
+CONFIG_THREAD_CUSTOM_DATA=n
+CONFIG_FLOAT=n
+CONFIG_BOOT_BANNER=n
+CONFIG_BOOT_DELAY=0
+CONFIG_CONSOLE=n
+CONFIG_UART_CONSOLE=n
+CONFIG_STDOUT_CONSOLE=n
+CONFIG_PRINTK=n
+CONFIG_EARLY_CONSOLE=n
```

Comment:
@doki-nordic shall `CONFIG_BOARD_ENABLE_CPUNET` be added as well now?

Reply:
Yes, I added it today.